libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -303,6 +305,16 @@ void AP_MotorsMulticopter::update_throttle_filter()
     } else {
         _throttle_filter.reset(0.0f);
     }
+
+    float new_thr = _throttle_filter.get();
+
+    if (!is_equal(last_thr, new_thr)) {
```

Comment:
this is a bit odd? Why not update if there equal?

Reply:
I think this is because when the throttle filter is off, manual updates are at 50 Hz so it is constant the majority of the time and I could see that making slew rate difficult to calculate.

Reply:
without this the output ends up recording zero slew a lot of the time and the system doesn't work (and the slew bounces up and down the whole time)